packages/spanner/system-test/spanner.js

```diff
@@ -396,6 +396,10 @@ describe('Spanner', function() {
         done();
       });
     });
+
+    it('should keep the session alive', function(done) {
+      session.keepAlive(done);
```

Comment:
Is there a more elaborate way to test that it's working at keeping the session alive? For example, if a session normally expires after `x` ms, can we test that keepAlive keeps a session still alive after `x` ms?